pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -614,6 +614,15 @@ def hidewhen_not_tahoe(render_setting_categories):
                 'defaultValue': True
             }
         ]
+    },
+    {
+        'name': 'ImageTransformation',
+        'settings': [
+            {
+                'name': 'forceVerticalFlip',
```

Comment:
Not sure about the name here.  Why "forceVerticalFilp"

Why not just "flipY" or "flipVertical"

Reply:
yep, sounds better, updated